server/Cargo.toml

```diff
@@ -18,3 +18,4 @@ tokio = { workspace = true, features = ["full"] }
 tracing = "0.1"
 tracing-subscriber.workspace = true
 zbus.workspace = true
+zeroize.workspace = true
```

Comment:
This is no longer used

Reply:
Sorry, I don't understand. Also, [this](https://github.com/bilelmoussaoui/oo7/pull/164#discussion_r1864965430)

Reply:
As mentioned before, you don't to use zeroize in that specific case but it is okay.